startup/csx1/startup/settings.py

```diff
@@ -35,13 +35,16 @@
     # getattr(sclr.channels, f'chan{i}').kind = 'normal' will remove the
     # hinted fields from LivePlot and LiveTable.
 
+
+
 def relabel_fig(fig, new_label):
     fig.set_label(new_label)
     fig.canvas.manager.set_window_title(fig.get_label())
 
 # fccd.hints = {'fields': ['fccd_stats1_total']}
 for i in [1, 2, 3, 4, 5]:
     getattr(fccd, f'stats{i}').total.kind = 'hinted'
+fccd.mcs.read_attrs = fccd.mcs.read_attrs[0:7] #silences the channels we do not use (7-32)
```

Comment:
Pre-line comments are generally easier to read when they are more than 1-2 words.

```suggestion
# Silence the channels we do not use (7-32)
fccd.mcs.read_attrs = fccd.mcs.read_attrs[0:7]
```